analyses/eval_analysis_spark.md

```diff
@@ -0,0 +1,115 @@
+
+### Eval and Function Analysis with Spark
+
+This is a continuation of the analysis on the usage of Eval and the Function class to create Javascript functions for web pages in the webcrawl dataset. This analysis uses Spark to look at the entire dataset, as apposed to the other analysis which looked at random samples in the dataset. The primary purpose of this analysis is to compare an analysis of the entire dataset with the previous analysis of small samples. This is being done to check if the small sample was representative of the entire dataset.  
+
+The previous Analysis can be found here: [eval_analysis.ipynb](eval_analysis.ipynb) and [eval_analysis.md](eval_analysis.md). An explanation for what eval is and a justification for why we are looking at eval usage can be found in the previous analysis.
```

Comment:
Nit: these links aren't working for me when viewing the notebook rendered on Github. I'd just use the full github.com URL to these files.